adapter-base/pom.xml

```diff
@@ -138,8 +142,8 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>org.bouncycastle</groupId>
-      <artifactId>bcpkix-jdk18on</artifactId>
+      <groupId>org.testcontainers</groupId>
```

Comment:
do we still need this?

Reply:
It is used for starting custom container with OCSP responder for integration testing. So I suppose we need to keep it here. https://github.com/kyberpunk/hono/blob/enable_revocation_check_for_trusted_ca/adapter-base/src/test/java/org/eclipse/hono/adapter/auth/device/x509/OCSPIntegrationTest.java#L64